game-core/src/main/java/games/strategy/sound/SoundOptions.java

```diff
@@ -44,23 +43,20 @@ private SoundOptions(final JComponent parent) {
     }
   }
 
-  /**
-   * Adds the "Sound Options" menu item to the specified menu.
-   *
-   * @param parentMenu menu where to add the menu item "Sound Options".
-   */
-  public static void addToMenu(final JMenu parentMenu) {
+  /** Builds a "Sound Options" menu item. */
+  public static JMenuItem buildSoundOptionsMenuItem() {
     final JMenuItem soundOptions = new JMenuItem("Sound Options");
     soundOptions.setMnemonic(KeyEvent.VK_S);
-    soundOptions.addActionListener(e -> new SoundOptions(parentMenu));
-    parentMenu.add(soundOptions);
+    soundOptions.addActionListener(e -> new SoundOptions(soundOptions));
```

Comment:
Looking at the class it really looks like the constructor should be replaced with a static method, because that's what it's actually being used as.
Arguably out-of-scope, but this is a real smell.

Reply:
https://github.com/DanVanAtta/triplea/pull/6

Reply:
So I updated my PR, the build should pass, I manually tested my changes, so yeah up to you really how you want to proceed